src/lib.rs

```diff
@@ -327,7 +327,6 @@ fn _assert_impls() {
     assert_sync::<Error>();
 
     assert_send::<Body>();
-    assert_sync::<Body>();
```

Comment:
Unfortunately this would be a breaking change :(

Reply:
Are you sure? I am not a Rust pro so I am probably wrong, but since I a removing a trait constraint to a parameter I was expecting this to be fine ?
Since all structs implementing TryStream+Send+Sync will implement TryStream+Send.

Reply:
> Since all structs implementing TryStream+Send+Sync will implement TryStream+Send.

Absolutely, but remember that `Body` isn't just an input for requests; it's also an output for response. Removing `Sync` from `Body` would disrupt users who rely on the body being `Sync`.